src/main/java/com/easypost/easyvcr/ExpirationActions.java

```diff
@@ -0,0 +1,19 @@
+package com.easypost.easyvcr;
+
+/**
+ * Enums representing different actions to take when a recording is expired.
+ */
+public enum ExpirationActions {
+    /**
+     * Warn that the recorded interaction is expired, but proceed as normal.
+     */
+    Warn,
+    /**
+     * Throw an exception that the recorded interaction is expired.
+     */
+    Throw_Exception,
```

Comment:
Question: The title case split by an underscore is a Java convention?...

Reply:
Convention is all caps, underlines for spaces: https://howtodoinjava.com/java/enum/enum-tutorial/#:~:text=By%20convention%2C%20enums%20are%20constants,(same%20as%20class%20names).

But I noticed that, oops, all the other enums in here are title cased like the C# version.

And then I just gave up at this weird hybrid...

Reply:
We can put down enum renaming as a breaking change for the future.

For now, the new enums with match the TitleCase format of the others.

Reply:
The beauty of a pre v1 release is we can break all we want. Let's do so separately, but let's also do this prior to going stable. Can you make a task to follow up?